File: drake_bazel_download/apps/find_resource_test.cc
// SPDX-License-Identifier: MIT-0

#include "drake/common/find_resource.h"
#include "drake/common/text_logging.h"

int main()
{
  drake::logging::set_log_level("debug");
  drake::FindResourceOrThrow("drake/examples/pendulum/Pendulum.urdf");
  return 0;
}

File: drake_bazel_external_legacy/apps/find_resource_test.py
# SPDX-License-Identifier: MIT-0

"""
Provides an example (and test) of finding resources with Python from a Bazel
project.
"""

import logging

from pydrake.common import FindResourceOrThrow

# If you have trouble finding resources, you can enable debug logging to see
# how `FindResource*` is searching.
logging.getLogger("drake").setLevel(logging.DEBUG)

FindResourceOrThrow("drake/examples/pendulum/Pendulum.urdf")
